shub/config.py

```diff
@@ -455,3 +480,12 @@ def get_version():
     """Load shub configuratoin and return version."""
     conf = load_shub_config()
     return conf.get_version()
+
+
+def list_targets(ctx, param, value):
```

Comment:
`list_targets` is used by `shub` and `shub.image` comands, and depends only on configuration file; let's move it here to avoid different import issues.